django_zbp/settings/production.py

```diff
@@ -2,7 +2,10 @@
 
 DEBUG = False
 
-STATICFILES_STORAGE = "whitenoise.storage.CompressedManifestStaticFilesStorage"
+try:
```

Comment:
I guess the django way would be more like:
```
STORAGES = {
    "default": {
        "BACKEND": "django.core.files.storage.FileSystemStorage",
    },
    "staticfiles": {
        "BACKEND": "whitenoise.storage.CompressedManifestStaticFilesStorage",
    },
}`
```
which seems a bit clearer to me than the try/except. What do you think?

Reply:
yes. I was not sure whether it might be imported from `base.py` in which case I might not want to override all of the `STORAGES`. But let's go with the full overwrite for `production.py`.

Reply:
ah, good point. I think it's ok to overwrite it though